src/Components/MainTableView/DataStorageClasses/TableAttribute.tsx

```diff
@@ -33,6 +33,11 @@ class TableAttribute {
      * @param timeString 
      */
     static parseTimeString(timeString: string) {
+      // Handle case with null
+      if (timeString === null) {
+        return 'null';
```

Comment:
How about we represent `null` throughout as:

```suggestion
        return '=NULL=';
```

That way it is similar to how blobs are represented.

Reply:
Sure, wasn't sure what was the datajoint covention was.

Reply:
Cool. There isn't a convention that is client universal but this would be a good place to use one taking note from how it was done for blob. Thanks :+1: